ui/lib/core/addon/templates/components/form-field.hbs

```diff
@@ -248,22 +248,40 @@
       {{/if}}
     {{else}}
       {{!-- Regular Text Input --}}
-      <input data-test-input={{attr.name}} id={{attr.name}} autocomplete="off" spellcheck="false"
-        value={{or (get model valuePath) attr.options.defaultValue}} onChange={{action
+      <input 
+        data-test-input={{attr.name}}
+        id={{attr.name}}
+        autocomplete="off"
+        spellcheck="false"
+        value={{or (get model valuePath) attr.options.defaultValue}}
+        onChange={{action
           (action "setAndBroadcast" valuePath)
           value="target.value"
-        }} class="input" maxLength={{attr.options.characterLimit}} />
-
+        }}
+        onkeyup={{action
+          (action "handleKeyUp" attr.name)
+          value="target.value"
+        }}
+        class="input {{if validationMessage "has-error-border"}}"
+        maxLength={{attr.options.characterLimit}} />
+      {{!-- Custom Validation --}}
       {{#if attr.options.validationAttr}}
         {{#if
           (and
             (get model valuePath) (not (get model attr.options.validationAttr))
           )
         }}
           <AlertInline @type="danger" @message={{attr.options.invalidMessage}} />
-
         {{/if}}
       {{/if}}
+      {{!-- Cp Validations Library --}}
```

Comment:
This isn't specific to the validations library, since anytime you pass in `validation message` this will render